packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5560,6 +5578,11 @@ class RawImage extends LeafRenderObjectWidget {
       ..filterQuality = filterQuality;
   }
 
+  @override
+  void didUnmountRenderObject(RenderImage renderObject) {
+    renderObject.image = null;
```

Comment:
Maybe add a comment here explaining why we need to set this to null.

Reply:
Done